lang/ucblogo/Portfile

```diff
@@ -1,38 +1,26 @@
-PortSystem 1.0
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-name             ucblogo
-version          6.2.2
-revision         0
-categories       lang
-maintainers      nomaintainer
-platforms        darwin
-description      Berkeley Logo
-long_description \
-    Berkeley Logo is a free Logo interpreter.
-homepage         https://people.eecs.berkeley.edu/~bh/logo.html
+PortSystem          1.0
+PortGroup           github 1.0
 
-# Note that license has changed to GPL-3 in 6.1
-license          GPL-3
+name                ucblogo
+github.setup        jrincayc ${name}-code 6.2.2 version_
+revision            1
```

Comment:
I don't think just increasing the `revision` field here will work. You do not change the `version` number, but do change the `master_sites` which caused the `checksums` to change. Therefore shouldn't this be considered a [stealth update](https://trac.macports.org/wiki/PortfileRecipes#stealth-updates)?

Reply:
Hi @reneeotten,

I am not sure what to do here - I originally updated the Portfile to use the version 6.2.2 tarball from the homepage of this project but @ryandesign correctly pointed out to me that this project had moved to github and that correct version 6.2.2 tarball should be pulled from github.  So I updated the Portfile accordingly.  If you have any suggestions on how to handle this, please let me know.

Thanks,

-ranga

Reply:
the problem here is that in this commit you do not change the versions but the checksums change; that will cause issues if my understanding is correct (@ryandesign feel free to correct me if I'm wrong!) So please read the section I linked above, that explains how to deal with this.

Reply:
Hi @reneeotten,

As recommended in the section you linked, I have added dist_subdir along with a note about removing dist_subdir when the next release comes out.  Hopefully this is the correct way to fix this issue.

Thanks,

-ranga

CC: @ryandesign